pom.xml

```diff
@@ -57,17 +57,18 @@
 			<scope>compile</scope>
 		</dependency>
 		<dependency>
-			<groupId>mysql</groupId>
-			<artifactId>mysql-connector-mxj-fixed</artifactId>
-			<version>5.0.12</version>
+			<groupId>ch.vorburger.mariaDB4j</groupId>
+			<artifactId>mariaDB4j</artifactId>
+			<version>2.4.0</version>
 			<scope>compile</scope>
 		</dependency>
 		<dependency>
-			<groupId>mysql</groupId>
-			<artifactId>mysql-connector-mxj-db-files-fixed</artifactId>
-			<version>5.0.12-20170418</version>
+			<groupId>ch.vorburger.mariaDB4j</groupId>
```

Comment:
@sherrif10 introducing maria db dependency only cannot make us run on maria db,we also need the maria db connector

Reply:
Thanks @herbertYiga, however mariadb can work fine with mysql connectors without even any modification

Reply:
@sherrif10  ohh yes,point noted.Could this be of use to you too https://mariadb.com/kb/en/mariadb-vs-mysql-compatibility/

Reply:
@dkayiwa  shared that to me earlier thanks again @herbertYiga